ehsm_kms_service/quote_manager_apis.js

```diff
@@ -0,0 +1,270 @@
+ *          ==> {r}policyId(String [36]) : quote policy ID. eg. '326f2049-56ab-474f-a1c0-516de230****'
+ * @returns {Object}
+ *          ==> {r}result(bool) : The result of quote verification. eg. 'true'
+ *          ==> {r}nonce(bool) : The nonce in BASE64 string. eg. 'bm9uY2U=
+ *          ==> {r}sign(String) : The HAMC sign of result and nonce calculated by the API Key. eg. 'T4DRCEZAPLBbb+d3ObD∗∗∗'
+ */
+const verifyQuote = async (res, appid, payload, DB, action) => {
+    try {
+        const { quote, nonce, policyId } = payload
+        if (!checkStringParam(quote, true, QUOTE_LENGTH_MAX)) {
+            res.send(_result(400, `quote cannot be empty, must be string and length not more than ${QUOTE_LENGTH_MAX}.`))
+            return
+        }
+        if (!checkStringParam(nonce, true, NONCE_LENGTH_MAX)) {
+            res.send(_result(400, `nonce cannot be empty, must be string and length not more than ${NONCE_LENGTH_MAX}.`))
+            return
+        }
+        if (!checkStringParam(policyId, true, UUID_LENGTH)) {
+            res.send(_result(400, `policyId cannot be empty, must be string and length equal ${UUID_LENGTH}.`))
+            return
+        }
+        if (!check_UUID_format(policyId)) {
+            res.send(_result(400, 'policyId format wrong.'))
+            return
+        }
+
+        // query quote policy by policyId
+        const query_quote_policy = {
+            selector: {
+                appid,
+                policyId
+            },
+            fields: ['mr_enclave', 'mr_signer'],
+            limit: 1
+        }
+        let quote_policy_res = await DB.partitionedFind('quote_policy', query_quote_policy)
+        if (quote_policy_res.docs.length != 1) {
+            res.send(_result(400, 'Invalid  policyId.'))
+            return
+        }
+        mr_enclave = quote_policy_res.docs[0].mr_enclave
+        mr_signer = quote_policy_res.docs[0].mr_signer
+        napi_res = napi_result(action, res, [quote, mr_signer, mr_enclave, nonce])
+        if (napi_res) {
+            let { error, hmac } = await gen_hmac(DB, appid, napi_res.result)
+            if (hmac.length > 0) {
+                napi_res.result.sign = hmac
+                res.send(napi_res)
+                return
+            } else {
+                res.send(_result(400, 'Internal error', {}))
+                return
+            }
+        }
+    } catch (e) {
+        console.info('verifyQuote :: ', e)
+        res.send(_result(500, 'Server internal error, please contact the administrator.'))
+        return
+    }
+}
+
+/**
+ * The UploadQuotePolicy Support uploading MRenclave and MRsigner and returning new policyid.
+ * @param {Object} res : response
+ * @param {String} appid : appid of user
+ * @param {Object} DB : database controller
+ * @param {Object} payload
+ *          ==> {r}mr_enclave(String [1~1024]) : A string of length 64. eg. '870c42c59bc74c7ad22869411709e4f78ac3c76add6693bb43296b03362e5038'
```

Comment:
same here, update the description of the mr_enclave and mr_signer

Reply:
will be change